client/wfprev-war/src/main/angular/src/app/components/edit-project/project-details/project-files/project-files.component.ts

```diff
@@ -34,52 +48,216 @@ export class ProjectFilesComponent {
     'delete'
   ];
 
-  // hardcode table, will be replaced by the attachment api data
-  projectFiles = [
-    {
-      attachmentType: 'Activity Polygon',
-      fileName: 'Polygon234',
-      fileType: 'KMZ',
-      uploadedBy: 'IDIR\\LULI',
-      uploadedDate: '2024-12-05',
-      polygonHectares: 22555,
-      description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit.'
+  ngOnInit(): void {
+    this.loadProjectAttachments();
+  }
+
+  loadProjectAttachments(): void {
+    if (this.projectGuid) {
+      this.attachmentService.getProjectAttachments(this.projectGuid).subscribe({
+        next: (response) => {
+
+          if (response?._embedded?.fileAttachment && Array.isArray(response._embedded.fileAttachment)) {
+            const fileAttachment = response._embedded.fileAttachment[0];
+
+            // Now, call getProjectBoundaries to get the boundary data
+            this.projectService.getProjectBoundaries(this.projectGuid).subscribe({
+              next: (boundaryResponse) => {
+                // Check if the response has the projectBoundary array and access the first boundary
+                const boundarySizeHa = boundaryResponse?._embedded?.projectBoundary?.[0]?.boundarySizeHa;
```

Comment:
Will there be any chances that there are multiple boundary attached to one project? sounds like we will only keep one and the first one?

Reply:
There shouldn't be as the API has been updated to upsert boundaries(create if one doesn't exist or update if one does). I added checks here just in case to return the boundaries/attachments with the latest timestamps instead of the first element in the array.